src/App.tsx

```diff
@@ -1,15 +1,97 @@
-import { ChakraProvider } from '@chakra-ui/react';
-import MealsList from 'components/Meals/MealsList/MealsList';
-import { theme } from 'others/theme';
-import AddSupplier from 'components/Supplier/AddSupplier';
-import Categories from 'components/Tables/Categories';
+import { ChakraProvider, ColorModeScript, Flex } from '@chakra-ui/react';
+
+import MainTemplate from 'components/templates/MainTemplate';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import { ROUTE_ACCESS, ROUTE_MENU, ROUTE_MENU_FORMS, ROUTE_SETTINGS, ROUTE_STATS, ROUTE_WAREHOUSE } from 'routing';
+import Employees from 'views/Access/Employees';
+import Dishes from 'views/Menu/Dishes';
+import Ingredients from 'views/Menu/Ingredients';
+import Products from 'views/Menu/Products';
+import General from 'views/Settings/General';
+import ABCAnalysis from 'views/Statistics/ABCAnalysis';
+import Customers from 'views/Statistics/Customers';
+import Payments from 'views/Statistics/Payments';
+import Sales from 'views/Statistics/Sales';
+import Taxes from 'views/Statistics/Taxes';
+import Stock from 'views/Warehouse/Stock';
+import Subscription from 'views/Settings/Subscription';
+import Supplies from 'views/Warehouse/Supplies';
+import Tables from 'views/Settings/Tables';
+import DishForm from 'views/Menu/DishForm';
+import ProductForm from 'views/Menu/ProductForm';
+import IngredientForm from 'views/Menu/IngredientForm';
+import Categories from 'views/Menu/Categories';
+import CategoriesForm from 'views/Menu/CategoriesForm';
 
 const App = () => (
-  <ChakraProvider theme={theme}>
-    <AddSupplier />
-    <Categories />
-    <MealsList />
-  </ChakraProvider>
+  <Router>
+    <ChakraProvider>
+      <ColorModeScript initialColorMode="light" />
+      <MainTemplate>
+        <Switch>
+          <Flex alignItems="center" flexDir="column" justifyContent="center">
+            <Route path={ROUTE_STATS.SALES.path}>
```

Comment:
You can think of making it more generic as you did in other files.
You can define `component` property typed as `React.ReactNode` in each `Route` like this:
```
export const ROUTE_SETTINGS: Route<IRouteData> = {
  GENERAL: { path: '/settings/general', name: 'Ogólne', component: General },
  BILLING: { path: '/settings/billing', name: 'Subskrypcja', component: Subscription },
  TABLES: { path: '/settings/tables', name: 'Stoliki', component: Tables },
};
 ```

```
export interface IRouteData {
  path: string;
  name: string;
  readonly component: React.ReactNode;
}
```

Then you can export `[...ROUTE_STATS, ...ROUTE_MENU, ...].map(route => ({route.path, route.component})`, access it here and map to HTML tags:
```
{
  routes.map(route => {
    return (
      <Route path={route.path}>
         {children}
       </Route>
     );
  })
}
```